tests/test_flows.py

```diff
@@ -81,8 +81,21 @@ def tcpdump_start(self):
         if opts.tcpdump:
             outputfile = os.path.join(self.rootdir, 'logs', 'test_{}.pcap'.format(self.running))
             debug("Starting tcpdump, output file {}".format(outputfile))
-            inspect = json.loads(subprocess.check_output(('docker', 'network', 'inspect',
-                                                          'prplMesh-net-{}'.format(self.opts.unique_id))))
+            docker_network = 'prplMesh-net-{}'.format(self.opts.unique_id)
+            docker_network_inspect_cmd = ('docker', 'network', 'inspect', docker_network)
+            inspect_result = subprocess.run(docker_network_inspect_cmd, stdout=subprocess.PIPE)
+            if inspect_result.returncode != 0:
+                # Assume network doesn't exist yet. Create it.
+                # This is normally done by test_gw_repeater.sh, but we need it earlier to be able to
+                # start tcpdump
+                # Raise an exception if it fails.
+                subprocess.run(('docker', 'network', 'create', docker_network), check=True,
+                               stdout=subprocess.DEVNULL)
+                # Inspect again, now raise if it fails.
```

Comment:
I might be missing something here, but where is the `raise Exception` ?

Reply:
It's in the `check = True`.